packages/interactivity-router/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
 	"$schema": "https://json.schemastore.org/tsconfig.json",
-	"extends": "../../tsconfig.base.json",
-	"compilerOptions": {
-		"checkJs": false,
-		"strict": false
-	},
-	"references": [ { "path": "../a11y" }, { "path": "../interactivity" } ]
+	"references": [
+		{ "path": "../a11y" },
+		{ "path": "../interactivity" },
```

Comment:
These don't need to be at this level, they're really dependencies of the main sub-project, but the linter requires them and they shouldn't be particularly harmful either.